include/dali/core/nvtx.h

```diff
@@ -100,6 +100,8 @@ struct DomainTimeRange : RangeBase {
 #endif
 };
 
+void SetThreadName(const char *name);
```

Comment:
Maybe some docs comment that only  first 15 characters of the name will be used?

Reply:
Done